sled-agent/src/services.rs

```diff
@@ -362,97 +514,31 @@ impl ServiceManager {
                 );
             }
 
-            // TODO: Remove once Nexus traffic is transmitted over OPTE.
-            let physical_nic = match req
-                .services
-                .iter()
-                .any(|service| matches!(service, ServiceType::Nexus { .. }))
-            {
-                true => {
-                    let vnic = self
-                        .physical_link_vnic_allocator
-                        .new_control(None)
-                        .map_err(|e| Error::NexusVnicCreation(e))?;
-                    Some(vnic)
+            let nic_needed = match self.nic_needed(req) {
+                Ok(n) => n,
+                Err(e) => {
+                    info!(self.log, "skipping zone {}: {:?}", req.zone_name, e);
```

Comment:
Should this be at `warn!` or `error!` log level? We may also want to specifically say that this is skipped because a TF-port variant zone requires the `/dev/tfport` device file.

Reply:
`warn!` would be a good idea.

The error message already tells you which zone isn't being created and why with "failed to find device {device}", which I think is all you're asking for.